src/webview/js/generatedEs3.js

```diff
@@ -551,9 +551,19 @@ documentBody.addEventListener('click', function (e) {
 
 var handleLongPress = function handleLongPress(target) {
   hasLongPressed = true;
+
+  if (target.closest('.reaction-list')) {
+    sendMessage({
+      type: 'reactionDetails',
+      messageId: getMessageIdFromNode(target),
+      targetReactionName: target.getAttribute('data-name') || ''
```

Comment:
Hmm -- what if `target` isn't the element with `data-name` but rather some descendant? For example, the `img` for an image emoji.

I think you want to get this from `target.closest('.reaction')`.